docs/user-guide/install.md

```diff
@@ -158,3 +158,54 @@ Windows service vdvs uninstalled successfully!
 ```
 PS C:\> Set-ExecutionPolicy Unrestricted -Scope Process -Force
 ```
+
+## Upgrade Instructions
+
+The volume driver can be upgraded or reinstalled any time with no impact to the user data. User data in the volumes stays intact and remains accessible post upgrade.
```

Comment:
Upgrade of Guest Plugin or ESX Server has no impact on data path and running containers. It is recommend to have Guest Plugin and ESX Server from same release. However, it is not mandatory, Plugin and ESX Server version are backward compatible.